sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala

```diff
@@ -20,14 +20,21 @@ package org.apache.spark.sql.execution.datasources.jdbc
 /**
  * Options for the JDBC data source.
  */
-private[jdbc] class JDBCOptions(
+private[sql] class JDBCOptions(
```

Comment:
Actually, if my understanding is correct, we don't need `private[sql]` here. (See  https://github.com/apache/spark/commit/511f52f8423e151b0d0133baf040d34a0af3d422)